### DISCUSSION TITLE
Is there a way to show corfu-popupinfo-documentation for a condidate after its completion is filled?

### DISCUSSION THREAD

**Discussion by IceAsteroid** [minad/corfu#444] [Q&A] [answered]
It's not always good to delete parts of the completion after it's finished, in order for corfu to show popup info for the selected candidate.

And also, sometimes there just only needs to show the brief information of the candidate and not to open its documentation by ```C-h v```, ```C-h f```, or ```C-h o``` in another window.

How could I do this? If I did ```corfu-popupinfo-documentation```, it would complaint ```No candidate selected```.

**Comment by minad** [comment]
I don't understand the question. Please clarify *in detail*.

**Reply by IceAsteroid** [reply]
To show corfu-popupinfo-documentation's popup after a candidate is selected in the corfu's completion popup.

Which means the completion is filled and closed, can I still show the corfu-popupinfo-documentation's popup for the keyword that the cursor is currently at?

For example: (setq-local ... ...) is completed from a corfu completion. Can I still show the documentation's popup for the keyword "setq-local" after it's selected and the corfu completion popup is closed?

**Reply by minad** [reply, marked as answer]
Thanks for the explanation. No, that's not possible with Corfu. You can use something like Eldoc, Xref or Embark to show the documentation of the identifier at point.
